tools/auto-compose/Pipfile.lock

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
remove Pipfile.lock and add to your gitignore. This file should always be generated by developer using Pipfile.